qualification/tied_array_channelised_voltage/test_linearity.py

```diff
@@ -70,6 +70,10 @@ async def _test_linearity(
     assert scales[middle] == pytest.approx(1.0)
     powers = np.zeros_like(scales)
     for batch in batched(enumerate(scales), len(receiver.stream_names)):
+        # The last batch might be short (since the number of scales won't be
+        # multiple of the number of streams. When zipping with streams, we
+        # don't expect the iterables to be the same length, and some streams
+        # will be unused.
```

Comment:
Please and thank you. I have to unfortunately flag two things
1. I don't see a closing bracket for the second point in the first sentence
2. Adding to that, "since the number of scales won't be *a multiple [...]" ?

Reply:
Thanks. Guess I shouldn't have staying up working so late last night :-/